config.py

```diff
@@ -1,25 +1,27 @@
 import os
-user = "root"
-password = "root"
-host = "localhost"
-database = "mapbot"
-port = "3306"
-key = "*Google_Cloud_API_key*"  # Will be provided by mentors
+import dotenv
 
-# your_path_to_stanford-corenlp-x.x.x.jar
-stanford_path_to_jar = "./stanford-corenlp-full-2018-10-05/stanford-corenlp-3.9.2.jar"
+dotenv.load_dotenv()
 
-# your_path_to_stanford-corenlp-x.x.x-models.jar
-stanford_path_to_models_jar = "./stanford-corenlp-full-2018-10-05/stanford-corenlp-3.9.2-models.jar"
+user = os.getenv('mapbotuser')
+password = os.getenv('mapbotpassword')
+host = os.getenv('mapbothost')
+database = os.getenv('mapbotdatabase')
+port = os.getenv('mapbotport')
+key = os.getenv('mapbotkey')
+
+stanford_path_to_jar = os.getenv('stanford_path_to_jar')
+
+stanford_path_to_models_jar = os.getenv('stanford_path_to_models_jar')
+
+javahome = os.getenv('javahome')
 
-# for eg. 'C:\\Program\ Files\\Java\\jdk1.8.0_201\\bin\\java.exe' or '/usr/local/openjdk-11/bin/java'
-javahome = '*your_path_to_jdk_bin_java.exe*'
 
 # DONOT CHANGE THE VALUES BELOW DURING INITIAL CONFIGURATION SET UP
 
 if os.getenv("DOCKER"):
     # print("Inside Docker")
-    user = "root"
-    password = "root"
-    host = "db"
-    javahome = '/usr/local/openjdk-11/bin/java'
+    user = os.getenv('dockeruser')
+    password = os.getenv('dockerpassword')
+    host = os.getenv('dockerhost')
+    javahome = os.getenv('dockerjavahome')
```

Comment:
@chttrjeankr, would we need these variables separately in the docker implementation now?

Reply:
Not exactly. I plan to use `docker.env` as a separate environment file, so as to simplify things. Based on the current structure though, at the time of this PR, I believe this implementation would be okay to proceed with.